tests/conftest.py

```diff
@@ -33,8 +32,8 @@ def paths_by_extension(extension):
     """ Return a list of files in the examples directory that has a particular extension. """
 
     paths = []
-    for filename in os.listdir(examples_dir):
+    for filename in os.listdir(EXAMPLES_DIR):
         if fnmatch.fnmatch(filename, "*." + extension):
-            paths.append(examples_dir + "/" + filename)
+            paths.append(EXAMPLES_DIR + "/" + filename)
```

Comment:
This is guarantees failing cross-platform compatibility. Isn't there a constant for for path separator, or a function that joins paths in the manner appropriate for the OS it's running on?